contracts/WildcardSteward_v2.sol

```diff
@@ -158,6 +161,7 @@ contract WildcardSteward_v2 is Initializable {
             assetToken.mintWithTokenURI(address(this), tokens[i], tokenUri);
             benefactors[tokens[i]] = _benefactors[i];
             state[tokens[i]] = StewardState.Foreclosed; // TODO: Maybe Implement reverse dutch auction on intial sale or other such mechanisms to avoid the deadloss weight of
+            price[tokens[i]] = 10**17; // set intial price to 0.1 eth
```

Comment:
Setting the initial price to 0.1 eth of any tokens added. We can definitely configure this better. Whenever we are in the foreclosure state, price[tokenId] becomes the currently available auction price that it can be purchased at. We should do some thinking on how this amount can slide over time. Although I have my reservations on an ever decreasing price. Think psychologically, it makes people wait too much.

Reply:
Yeah, what I have in my mind is that it starts really high, that way we have the best chance of raising more funds, we can also set a minimum price. Maybe we should add something nice like a "first owner badge" or something, so that people are incentivized to buy it immediately.

> Think psychologically, it makes people wait too much.

I think on non-always for sale items that people actually want this isn't really true (especially if you have had a few items you wanted swiped from under your nose before because of waiting too long). Maybe with always for sale items it is different unless we can give some prestige to the first buyer.

Another facet is that if/when we have an algorithmic/manket based release schedule (eg, when all tokens are worth more than 0.1 eth release a new wildcard), it might make sense to have a fixed starting price.

Reply:
I would probably for now make this value an imput parameter, like `startingPrices`

Reply:
Awesome! Yeah I think at the moment lets not bother with the algorithmic release, I think the weekly cadence is better imo. 

I really like the idea of a first owner badge or something really hectic/special that incentivizes the owner. We should also make it fairly explicit that the money on that sale goes directly toward building and maintaining the wildcards platform which should be a plus.

Reply:
Are you aware of anyway to have the price of the asset follow some path and decrease from a max to a min over time? Not sure how to implement this in solidity and haven't seen anything useful so far. This is the missing part that needs implementation, then I think we are good to test.

Reply:
I'm guessing we create some equation as we use the block timestamp, and over time, the price value would decrease as now increases. Would need to adjust the slippage function to be more lenient if we do this ever moving implementation

Reply:
If you think about it is very similar to how the deposit decreases over time, in that case the slope is defined by the price of the token and patronage rate of the token.

I think for the auction we don't have a slippage function. But we keep `buy` and `buyAuction` as two different functions so that if someone does get front-run the tx just reverts (they don't lose any money). - I guess this is just an old comment